test/core/primitives/bitvec/bitvec_test.cpp

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "primitives/bitvec/bitvec.hpp"
+
+#include <gtest/gtest.h>
+
```

Comment:
Place tests in `namespace fc::primitives`

Reply:
Fixed